imctools/scripts/exportacquisitioncsv.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+import argparse
+import os
+import pandas as pd
+import imctools.io.mcdxmlparser as mcdmeta
+
+SUFFIX_ACMETA = '_' + mcdmeta.ACQUISITION + mcdmeta.META_CSV
+COL_MCD_ID = mcdmeta.ID
+COL_ACID = mcdmeta.ACQUISITIONID
+COL_ACSESSION = 'AcSession'
+SUF_CSV = '.csv'
+AC_META = 'acquisition_metadata'
+
+VAL_TEMP = 'temp'
+"""
+Reads all the acquisition metadata from the ome folders and concatenates them
+to a csv that contains all the metadata.
+"""
+
+def _read_and_concat(fol_ome, suffix, idname):
+    ac_names = os.listdir(fol_ome)
+    dat = pd.concat([pd.read_csv(os.path.join(fol_ome, a, a+suffix)) for a in ac_names],
```

Comment:
What's the purpose of first creating an index and then dropping it? Please clarify (comment in code)

Reply:
You are write, this can indeed be written more simply.